cmsl1t/recalc/met.py

```diff
@@ -2,8 +2,27 @@
 import numpy as np
 
 
+class met(object):
```

Comment:
Please capitalise classes. Probably needs to be moved to a different location later.